apps/docs/content/guides/database/extensions/pgaudit.mdx

```diff
@@ -41,68 +46,389 @@ To disable an extension you can call `drop extension`.
 </TabPanel>
 </Tabs>
 
-## Settings
+## Extension Configuration
+
+pgaudit can be configured to focus on multiple different levels of precision
+
+**pgaudit Logging Precision:**
```

Comment:
the word "precision" isn'tused in pgaudit docs, would want to be cautious of introducing terminology, maybe just saying "different logging modes"

Reply:
I'll change it to modes